src/analysis/normed_space/add_torsor.lean

```diff
@@ -37,7 +37,8 @@ variables (V : Type u) {P : Type v} [normed_group V] [metric_space P] [normed_ad
 include V
 
 /-- The distance equals the norm of subtracting two points. This lemma
-is needed to make V an explicit rather than implicit argument. -/
+is needed to make V an explicit rather than implicit argument. Without
+`V` as an explicit argument, `rw dist_eq_norm_vsub` does not work. -/
```

Comment:
The phrasing is confusing here. Can you change this to

"In this lemma, it is necessary to have `V` as an explicit argument; otherwise `rw dist_eq_norm_vsub` sometimes doesn't work."

Reply:
(It currently reads as if we need to prove this lemma, so that somewhere else it is possible to make `V` an explicit argument.)

Reply:
Rephrased as requested.